app/Models/Contact/Tag.php

```diff
@@ -51,6 +51,7 @@ public static function contactsCount()
                     ->join('tags', 'tags.id', '=', 'contact_tag.tag_id')
                     ->where('tags.account_id', auth()->user()->account_id)
                     ->groupBy('tag_id')
+                    ->orderBy('name')
```

Comment:
Actually the `orderBy` sql instruction is really bad when it comes to non-ascii characters.
You'd better order the list after the get, with the `->sortByCollator` `Collection` macro.

Reply:
Thanks for the info! Resolved in https://github.com/monicahq/monica/blob/72ebeb5eb4dd1aea1a33f02b2b328263060f35fe/app/Models/Contact/Tag.php#L55